typedoc.json

```diff
@@ -3,5 +3,6 @@
     "src/index.ts"
   ],
   "excludeTags": ["internal"],
-  "out": "docs/public"
+  "excludeInternal": true,
```

Comment:
with this being the right setting, I don't think we should keep the line above this one to `"excludeTags": ["internal"]`, since presumably they wouldn't be present in the first place, and if we somehow accidentally lose this flag, excluding the internal tag from being displayed would just make it more confusing to our users